js/src/main/scala/org/portablescala/reflect/Reflect.scala

```diff
@@ -36,12 +41,15 @@ object Reflect {
    *  @param loader
    *    Ignored
    */
+  @silent(
+      "parameter value loader in method lookupLoadableModuleClass is never used")
```

Comment:
Oh, I see. Yeah, that's a bit more convoluted :P is it possible to attach the annotation on the parameter itself?

Also: The text should probably explain *why* it is never used. Basically some slur about this needing to provide the exact same API surface than for the JVM.

Reply:
Oh, great idea! I didn't know one can annotate individual parameters :+1: 
(The text has to match the message from the compiler.)